src/components/lock/LockedDetails.tsx

```diff
@@ -78,7 +78,7 @@ const LockedDetails = ({
       if (differenceInDays > 0) setDaysDiff(differenceInDays)
       else setDaysDiff(0)
     }
-  }, [lockEndDate])
+  }, [lockEndDate, daysDiff])
```

Comment:
Kindly confirm its not causing unnecessary re-rendering

Reply:
Yeah, I’ll check. I added it to remove the dependency warnings.